lib/opal/nodes/closure.rb

```diff
@@ -223,11 +225,13 @@ def compile_catcher
             end
             line "throw $e;"
           end
-          line "}"
+          line "} finally {", *catchers_without_eval_return.map { |type| "$t_#{type}.is_orphan = true;" }, "}"
```

Comment:
Can we skip the finally clause if `catchers_without_eval_return` is empty?

Reply:
Yes, I changed the code.